scripts/Inelastic/CrystalField/energies.py

```diff
@@ -0,0 +1,43 @@
+from mantid.simpleapi import *
```

Comment:
The `import *` import here is overkill. We should limit it to `CrystalFieldEnergies`